dev-requirements.txt

```diff
@@ -12,3 +12,8 @@ Sphinx==1.6.5
 sphinx_rtd_theme
 -e git+https://git@github.com/uc-cdis/cdisutils-test.git@0.0.1#egg=cdisutilstest
 -e git+https://git@github.com/NCI-GDC/signpost.git@c8e2aa5ff572c808cba9b522b64f7b497e79c524#egg=signpost
+# these two are required by fence -> cirrus -> google_cloud
```

Comment:
should these be in dev-requirements or requirements?

Reply:
requirements..

Reply:
oh actually just dev-requirements, fence is only needed for tests